packages/commons/src/conditionals/validate.ts

```diff
@@ -510,6 +510,14 @@ export function getValidatorsForField(
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const jsonSchema = validator as any
 
+      /*
+       * In this case it was possible the validator is a or(...) and(...) or similar combinator value
+       * From these it's a little tricky to extract the field specific validation
+       */
+      if (!jsonSchema.properties) {
+        return null
+      }
+
```

Comment:
So we won't get validations in the name subfields?

Reply:
We never did if they were wrapped in `or` or `and` as the `anyOf` JSON schema validator does not have a `properties` field.

Currently the function expects an object validator from which we pick the branches

```
firstname: aValidator
middlename: bValidator
lastname: cValidator
```

so lastname would pick `cValidator`. It gets a little more complex if you have

```
(firstname: aValidator) OR (middlename: bValidator) OR (lastname: cValidator)
```

let alone something more nested so I left it unimplemented for now simply because of time constraints